packages/material-ui-system/src/propsToClassKey.js

```diff
@@ -1,4 +1,6 @@
-import { unstable_capitalize as capitalize } from '@material-ui/utils';
+function capitalize(string) {
```

Comment:
TODO: convert to import from utils, once https://github.com/mui-org/material-ui/pull/26485#issuecomment-850343427 is fixed